packages/create-miniapp-cli/src/installers/helm/index.ts

```
@@ -0,0 +1,55 @@
+import { AppType } from '../../consts.js'
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)